assets/javascripts/builder.jsx

```diff
@@ -0,0 +1,35 @@
+var React = require('react');
+var riek = require('riek')
```

Comment:
Gonna get nit picky with your JS now, most people prefer to combine multiple `var` statements into one seperated with commas. It's purely a style thing, but seems to be a best practice.

```js
var React = require('react'),
    riek = require('riek'),
    RIEInput = riek.RIEInput,
    FormElement = require('./formElement.jsx');
```